tablet/cl_main.lua

```diff
@@ -124,11 +124,11 @@ function DisplayModule(module, show)
 end
 
 -- Set Module URL (for iframes)
-function SetModuleUrl(module, url)
+function SetModuleUrl(module, url, hasComID)
 	DebugMessage("sending url update message to nui", module)
 	SendNUIMessage({
 		type = "setUrl",
-		url = url,
+		url = url + hasComID and ("&cachebuster=" + GetGameTimer()) or ("?cachebuster=" + GetGameTimer()),
```

Comment:
This gets the in game time, which repeats itself rather often. please use a function to get system time in UNIX timestamp format. This will ensure the value is always unique and never used before.